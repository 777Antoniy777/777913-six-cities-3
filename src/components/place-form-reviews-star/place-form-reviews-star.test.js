import React from 'react';
import renderer from "react-test-renderer";
import PlaceFormReviewsStar from "./place-form-reviews-star";

// set mocha data
const star = {
  id: 1,
  title: `perfect`,
};
const rating = false;

const handleInputChange = () => {};

it(`render PlaceFormReviews`, () => {
  const tree = renderer.create(
      <PlaceFormReviewsStar
        star={star}
        rating={rating}
        handleInputChange={handleInputChange}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
