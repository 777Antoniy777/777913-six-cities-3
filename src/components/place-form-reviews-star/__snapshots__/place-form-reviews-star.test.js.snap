// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PlaceFormReviews 1`] = `
Array [
  <input
    checked={false}
    className="form__rating-input visually-hidden"
    id="1-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={1}
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="1-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height={33}
      width={37}
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
]
`;
